src/core/card/LargeCard.stories.tsx

```diff
@@ -24,7 +24,7 @@ const Template: ComponentStory<typeof LargeCard> = (args) => (
     }}
   >
     <div style={{ maxWidth: 1470, margin: 24 }}>
-      <LargeCard {...args} />
+      <LargeCard {...args} url="https://hel.fi" hasLink />
```

Comment:
if there is an url parameter, is the hasLink needed?

Reply:
has link is just showing arrow